src/main/resources/org/sagebionetworks/web/client/view/SearchViewImpl.ui.xml

```diff
@@ -57,7 +57,7 @@
         <bh:Div addStyleNames="col-md-3 col-md-pull-9">
           <bh:Italic>Searching for a user profile? Try our</bh:Italic>
           <b:Anchor
-            href="#!PeopleSearch:"
+            href="PeopleSearch:"
```

Comment:
Based on some local testing, I think that the `href` needs a forward slash for the path to be relative to the current URL -- 

```suggestion
            href="/PeopleSearch:"
```

without slash | with slash
:---:|:---:
<img width="630" alt="image" src="https://github.com/Sage-Bionetworks/SynapseWebClient/assets/26949006/8fd40cde-dcf0-4999-91ef-8ec834da0144"> | <img width="690" alt="image" src="https://github.com/Sage-Bionetworks/SynapseWebClient/assets/26949006/8d6cfe01-41d2-4de6-aabb-6015606c4641">